### DISCUSSION TITLE
Possible to update MHI2 firmware from another unit?

### DISCUSSION THREAD

**Discussion by davephoto** [Mr-MIBonk/M.I.B._More-Incredible-Bash#204] [Q&A]
I have a MY2016 VW Golf R (NAR, Canada) with factory Discover Pro (MHI2_US_VWG11_S2126). I've been able to activate the Sport HMI and update to 2022/2023 maps thanks to these resources. 

Recently, I helped a friend with a similar car — MY2016, but slightly different MHI2_US_VWG11_P2126_1 — with the same modifications. The Sport HMI in this vehicle has additional displays (e.g. power in kW) that are absent in my car.

I suspect this might be due to the slight difference in MHI2 firmware between our cars. Is it possible to use More Incredible Bash to pull the firmware from the newer P2126_1 unit and flash it onto my S2126?

**Comment by harman-f** [comment]
More liekly relatet to coding or different ECU...

**Reply by davephoto** [reply]
Thanks for the insight. I will investigate further.

**Comment by LKSMNT** [comment]
@davephoto I'm also in Canada with MY2016 VW GTI. SW Train is same as your friend. (MHI2_US_VWG11_P2126_1)  I am stuck on maps from 2016 and would really like to update. Did you update maps by adding a lifetime FEC code? Need some help here haha..

**Reply by davephoto** [reply]
I was able to update my maps with the P87_N60S5MIBH3_NAR_NT_Q3.21_2022-23 data. I played around and guessed at different map eligibility FECs. I found that 093100EE allowed the update to take.
